src/test/cypress/integration/ModelingExercise.spec.ts

```diff
@@ -59,16 +56,17 @@ describe('Modeling Exercise Spec', () => {
 
         after('delete Modeling Exercise', () => {
             cy.login(admin);
-            cy.deleteModelingExercise(modelingExercise.id);
+            courseManagementRequests.deleteModelingExercise(modelingExercise.id);
         });
 
         it('Create a new modeling exercise', () => {
+            cy.intercept('POST', '/api/modeling-exercises').as('createModelingExercise');
             cy.visit(`/course-management/${testCourse.id}/exercises`);
             cy.get('#modeling-exercise-create-button').click();
-            cy.get('#field_title').type('Cypress Modeling Exercise');
-            cy.get('#field_categories').type('e2e-testing');
-            cy.get('#field_points').type('10');
-            cy.contains('Save').click();
+            createModelingExercise.setTitle('Cypress Modeling Exercise');
+            createModelingExercise.addCategories(['e2e-testing', 'test2']);
+            createModelingExercise.setPoints(10);
+            createModelingExercise.save();
             cy.wait('@createModelingExercise').then((interception) => {
                 modelingExercise = interception?.response?.body;
             });
```

Comment:
I would suggest moving the `cy.wait()` into the save method and return the chainable. Your test can then do something like this:
```typescript
createModelingExercise.save().then((interception) => {
    modelingExercise = interception?.response?.body;
});
```

Reply:
i did this originally but it did let to some other problems so i would like to keep it like it is.